tokio-util/src/lib.rs

```diff
@@ -1,4 +1,4 @@
-#![doc(html_root_url = "https://docs.rs/tokio-util/0.6.2")]
+#![doc(html_root_url = "https://docs.rs/tokio-util/0.6.3")]
```

Comment:
Btw, api-guidelines no longer recommends `html_root_url` (https://github.com/rust-lang/api-guidelines/issues/229). We can probably remove `html_root_url`.

Reply:
If we want to remove it, we can have a PR that takes it out of every Tokio crate at once.

Reply:
That makes sense. I created #3489.